src/pages/groupComparison/styles.module.scss

```diff
@@ -14,14 +14,9 @@
         overflow:auto;
         padding:3px;
 
-        :global(div.groupItem) {
-            margin-bottom:6px;
-        }
-        :global(div.groupItem):first-child {
-            margin-top:0px;
-        }
-        :global(div.groupItem):last-child {
+        :global(div.checkbox) {
             margin-bottom:0px;
+            margin-top:0px;
         }
 
         :global(.markedForDeletion) {
```

Comment:
i know this isn't part of this PR but seems like maybe "markedForDeletion" would be a good to not have global here.    I guess maybe groupItem is another.

Reply:
kk. I actually got rid of groupItem altogether